templates/course_edit.html

```diff
@@ -120,7 +120,7 @@ <h3>
           <h3>
             <span class="txt">Course info.</span>
           </h3>
-          <form action="/course/{{ course.slug }}/edit" method="post" id="course-edit-form">
+          <form action="/course/{{ course.slug }}/edit" method="post" id="course-edit-form" data-fit-wysiwyg data-fit-wysiwyg-id="course_summary">
```

Comment:
What are you using `data-fit-wysiwyg-id` for?

Reply:
Missed that one.